src/core/modules/impl/ContractDefinitionLoader.ts

```diff
@@ -40,7 +40,7 @@ export class ContractDefinitionLoader implements DefinitionLoader {
   async doLoad<State>(contractTxId: string, forcedSrcTxId?: string): Promise<ContractDefinition<State>> {
     const benchmark = Benchmark.measure();
 
-    const contractTx = await this.arweaveWrapper.tx(contractTxId);
+    const contractTx = await this.arweave.transactions.get(contractTxId); //await this.arweaveWrapper.tx(contractTxId);
```

Comment:
@asiaziola small gift for you - https://github.com/redstone-finance/redstone-smartcontracts/issues/99